templates/sfn/ingest_sfn_definition.json.tpl

```diff
@@ -0,0 +1,153 @@
+{
+  "Comment": "${step_function_name}: A State machine for Judgment ingests",
```

Comment:
This Step Function will be for all ingests, maybe `State machine to ingest DR2 BagIt-like packages into Preservica.`